source/dotnet/Library/AdaptiveCards/Rendering/AdaptiveHostConfig.cs

```diff
@@ -25,14 +25,20 @@ public class AdaptiveHostConfig : AdaptiveConfigBase
         public FactSetConfig FactSet { get; set; } = new FactSetConfig();
 
         [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
-        public string FontFamily { get; set; } = "Segoe UI";
```

Comment:
> = "Segoe UI"; [](start = 46, length = 14)

This doesn't seem to be set anywhere anymore

Reply:
Moved to [AdaptiveHostConfig.cs](https://github.com/Microsoft/AdaptiveCards/pull/1988/files#diff-bd99de84a358191ba1ef5bdc1c0ad432R177).